core/app/controllers/uffizzi_core/api/cli/v1/account/credentials_controller.rb

```diff
@@ -40,6 +40,26 @@ def create
     respond_with credential_form
   end
 
+  # Update existing credentials of the given type
+  #
+  # @path [PUT] /api/cli/v1/account/credentials/{type}
+  #
+  # @parameter type(required,path) [string] Credentials type
+  # @parameter credential(required,body) [object<type:string>]
+  # @response 422 Unprocessable entity
+  # @response 200 OK
```

Comment:
We need to specify the actual response structure and generate Swagger docs based on this description (see the rakefile for the command to generate docs)